From 2448293126e61b337000d8844911cc1902acbe01 Mon Sep 17 00:00:00 2001
From: Colin Blundell <blundell@chromium.org>
Date: Mon, 25 Feb 2019 07:54:57 +0000
Subject: [PATCH] Remove FakeSigninManagerBase

After the conversion of the codebase to IdentityManager, this
functionality is no longer used.

Bug: 796544
Change-Id: Iad7695ed5b86f850c41d835aad45f6e9d4fef11b
Reviewed-on: https://chromium-review.googlesource.com/c/1482960
Commit-Queue: Colin Blundell <blundell@chromium.org>
Reviewed-by: David Roger <droger@chromium.org>
Cr-Commit-Position: refs/heads/master@{#635046}
---
 .../identity_test_environment_profile_adaptor.cc     |  5 ++---
 .../signin/core/browser/fake_signin_manager.cc       | 12 ------------
 components/signin/core/browser/fake_signin_manager.h | 10 ----------
 services/identity/identity_manager_impl_unittest.cc  |  2 +-
 .../identity/public/cpp/identity_manager_unittest.cc |  4 ++--
 .../identity/public/cpp/identity_test_environment.h  |  3 +--
 6 files changed, 6 insertions(+), 30 deletions(-)

diff --git a/chrome/browser/signin/identity_test_environment_profile_adaptor.cc b/chrome/browser/signin/identity_test_environment_profile_adaptor.cc
index b565582c58b96..e81af9ae3aac5 100644
--- a/chrome/browser/signin/identity_test_environment_profile_adaptor.cc
+++ b/chrome/browser/signin/identity_test_environment_profile_adaptor.cc
@@ -25,7 +25,7 @@ std::unique_ptr<KeyedService> BuildFakeSigninManagerForTesting(
   Profile* profile = static_cast<Profile*>(context);
   std::unique_ptr<SigninManagerBase> manager =
 #if defined(OS_CHROMEOS)
-      std::make_unique<FakeSigninManagerBase>(
+      std::make_unique<SigninManagerBase>(
           ChromeSigninClientFactory::GetForProfile(profile),
           ProfileOAuth2TokenServiceFactory::GetForProfile(profile),
           AccountTrackerServiceFactory::GetForProfile(profile));
@@ -150,8 +150,7 @@ IdentityTestEnvironmentProfileAdaptor::IdentityTestEnvironmentProfileAdaptor(
           static_cast<FakeProfileOAuth2TokenService*>(
               ProfileOAuth2TokenServiceFactory::GetForProfile(profile)),
 #if defined(OS_CHROMEOS)
-          static_cast<FakeSigninManagerBase*>(
-              SigninManagerFactory::GetForProfile(profile)),
+          SigninManagerFactory::GetForProfile(profile),
 #else
           static_cast<FakeSigninManager*>(
               SigninManagerFactory::GetForProfile(profile)),
diff --git a/components/signin/core/browser/fake_signin_manager.cc b/components/signin/core/browser/fake_signin_manager.cc
index 953ff28ddde75..728eb623afde4 100644
--- a/components/signin/core/browser/fake_signin_manager.cc
+++ b/components/signin/core/browser/fake_signin_manager.cc
@@ -12,18 +12,6 @@
 #include "components/signin/core/browser/signin_metrics.h"
 #include "components/signin/core/browser/signin_pref_names.h"
 
-FakeSigninManagerBase::FakeSigninManagerBase(
-    SigninClient* client,
-    ProfileOAuth2TokenService* token_service,
-    AccountTrackerService* account_tracker_service)
-    : SigninManagerBase(client, token_service, account_tracker_service) {}
-
-FakeSigninManagerBase::~FakeSigninManagerBase() {}
-
-void FakeSigninManagerBase::SignIn(const std::string& account_id) {
-  SetAuthenticatedAccountId(account_id);
-}
-
 #if !defined(OS_CHROMEOS)
 
 FakeSigninManager::FakeSigninManager(
diff --git a/components/signin/core/browser/fake_signin_manager.h b/components/signin/core/browser/fake_signin_manager.h
index a04d04e649087..e765f796d7761 100644
--- a/components/signin/core/browser/fake_signin_manager.h
+++ b/components/signin/core/browser/fake_signin_manager.h
@@ -15,16 +15,6 @@
 
 // SigninManager to use for testing.
 
-class FakeSigninManagerBase : public SigninManagerBase {
- public:
-  FakeSigninManagerBase(SigninClient* client,
-                        ProfileOAuth2TokenService* token_service,
-                        AccountTrackerService* account_tracker_service);
-  ~FakeSigninManagerBase() override;
-
-  void SignIn(const std::string& account_id);
-};
-
 #if !defined(OS_CHROMEOS)
 
 // A signin manager that bypasses actual authentication routines with servers
diff --git a/services/identity/identity_manager_impl_unittest.cc b/services/identity/identity_manager_impl_unittest.cc
index d0bb2b523f495..5b97f146e3eba 100644
--- a/services/identity/identity_manager_impl_unittest.cc
+++ b/services/identity/identity_manager_impl_unittest.cc
@@ -26,7 +26,7 @@ namespace identity {
 namespace {
 
 #if defined(OS_CHROMEOS)
-using SigninManagerForTest = FakeSigninManagerBase;
+using SigninManagerForTest = SigninManagerBase;
 #else
 using SigninManagerForTest = FakeSigninManager;
 #endif  // OS_CHROMEOS
diff --git a/services/identity/public/cpp/identity_manager_unittest.cc b/services/identity/public/cpp/identity_manager_unittest.cc
index f3f65d0a3d51e..182ed37e74ac5 100644
--- a/services/identity/public/cpp/identity_manager_unittest.cc
+++ b/services/identity/public/cpp/identity_manager_unittest.cc
@@ -44,7 +44,7 @@ namespace identity {
 namespace {
 
 #if defined(OS_CHROMEOS)
-using SigninManagerForTest = FakeSigninManagerBase;
+using SigninManagerForTest = SigninManagerBase;
 #else
 using SigninManagerForTest = FakeSigninManager;
 #endif  // OS_CHROMEOS
@@ -390,7 +390,7 @@ class IdentityManagerTest : public testing::Test {
 #if defined(OS_CHROMEOS)
     DCHECK_EQ(account_consistency, signin::AccountConsistencyMethod::kDisabled)
         << "AccountConsistency is not used by SigninManagerBase";
-    signin_manager_ = std::make_unique<FakeSigninManagerBase>(
+    signin_manager_ = std::make_unique<SigninManagerBase>(
         &signin_client_, &token_service_, &account_tracker_);
 #else
     signin_manager_ = std::make_unique<FakeSigninManager>(
diff --git a/services/identity/public/cpp/identity_test_environment.h b/services/identity/public/cpp/identity_test_environment.h
index eaa7e5cf0ad88..35ed37d10c42a 100644
--- a/services/identity/public/cpp/identity_test_environment.h
+++ b/services/identity/public/cpp/identity_test_environment.h
@@ -13,7 +13,6 @@
 class AccountTrackerService;
 class FakeAccountFetcherService;
 class FakeProfileOAuth2TokenService;
-class FakeSigninManagerBase;
 class FakeSigninManager;
 class GaiaCookieManagerService;
 class IdentityTestEnvironmentChromeBrowserStateAdaptor;
@@ -33,7 +32,7 @@ namespace identity {
 
 namespace {
 #if defined(OS_CHROMEOS)
-using SigninManagerForTest = FakeSigninManagerBase;
+using SigninManagerForTest = SigninManagerBase;
 #else
 using SigninManagerForTest = FakeSigninManager;
 #endif  // OS_CHROMEOS
